### ISSUE TITLE
Ctrl-DELETE is not recognised

### ISSUE THREAD

**Issue by facingBackwards** [thomasballinger/curtsies#104]
I am running a loop as shown in the examples:
```
with Input() as keys:
    for key in keys:
        print(key)
```
DELETE is recognised as expected but Ctrl-DELETE is not printed until the next character when it is shown as `^[[3;5~`. 

Adding the tuple `(b"\x1b[3;5~", u'<Ctrl-DELETE>')` to curtsieskeys.py fixes the problem for me but I'm not sure whether it'd cause problems elsewhere.

Let me know if there's a better way of handling this.
Thanks

**Comment by thomasballinger** [comment]
whoops tab took me to the wrong button first

**Comment by facingBackwards** [comment]
Should be fixed by #106
